doc/developer/trees.md

```diff
@@ -86,11 +86,11 @@ restatement of the fourth part of the definition).
 
 Most everything in mlpack is decomposed into a series of configurable template
 parameters, and trees are no exception.  In order to ease usage of high-level
-mlpack algorithms, each \c TreeType itself must be a template class taking three
+mlpack algorithms, each TreeType itself must be a template class taking three
```

Comment:
```suggestion
mlpack algorithms, each `TreeType` itself must be a template class taking three
```
The `\c` represents fixed-width (code) font in Doxygen, so the backticks can do that in Markdown. :)